sssom/constants.py

```diff
@@ -139,6 +142,21 @@
     RDF_SEE_ALSO,
 ]
 
+PREDICATE_FLIP_DICTIONARY = {
```

Comment:
Can we please externalize this mapping such that it can be re-used?

Reply:
Also such that it can be configured further without changing Python code.

Reply:
Yes we had already discussed it between us, but said to externlise in a future PR. I am less concerned with externalising (moving to a separately managed config file) the sssom default flip dictionary (I leave this up to @hrshdhgd to decide), but I think being able to override this from the user side is pretty important, so I requested the method to be extended.

Reply:
Done